### DISCUSSION TITLE
Nuked gopro 9 how to flash?

### DISCUSSION THREAD

**Discussion by igorYAKUT** [gopro/labs#569] [Q&A]
Friends, tell me if there is an opportunity to flash a goupro without an accomulator and a usb connector.

**Comment by igorYAKUT** [comment]
I have a goupro 9 lightweight without a back screen and a usb connector and accidentally installed the official version from the phone via Bluetooth and broke the phone

**Reply by dnewman-gpsw** [reply]
https://gopro.github.io/labs/install/

**Reply by igorYAKUT** [reply]
I understood it means there is no solution (((

**Reply by dnewman-gpsw** [reply]
The manual install will work on a camera with a SD card and battery only, doesn't require a screen or USB port.  With a "naked" camera, on stock firmware, you most use the original battery/cage for the update to work. Once Labs firmware is installed, the original battery is not a requirement.
